### PR TITLE
chore: Upgrade to Node 16

### PR DESCRIPTION
**Pull request by alanbsmith**
## Summary

Fixes: #2138

## Release Category

Infrastructure

---

## Checklist

- [ ] MDX documentation adheres to Canvas Kit's [standard MDX template](https://github.com/Workday/canvas-kit/discussions/1131)
- [ ] Label `ready for review` has been added to PR

## For the Reviewer

- [ ] PR title is short and descriptive
- [ ] PR summary describes the change (Fixes/Resolves linked correctly)
- [ ] PR Release Notes describes additional information useful to call out in a release message or removed if not applicable
- [ ] Breaking Changes provides useful information to upgrade to this code or removed if not applicable

## Areas for Feedback? (optional)

- [ ] Code
- [ ] Documentation
- [ ] Testing
- [ ] Codemods

### CODE REVIEW

File: modules/css/banner/lib/accessibility.scss [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 [data-whatinput='keyboard'] {
   @include wdc-focusable-banner() {
-    @extend .wdc-banner.wdc-banner-focus;
+    @extend .wdc-banner, .wdc-banner-focus;
```
[the remark this answers is not in the document]
Comment (mannycarrera4) [reply modules/css/banner/lib/accessibility.scss:11]: nvm, just saw your comment about selectors

**Review by mannycarrera4 — APPROVED** [review]

File: modules/css/banner/lib/accessibility.scss (hunk 1 above)
Comment (alanbsmith) [inline modules/css/banner/lib/accessibility.scss:11]:
```suggestion
    @extend .wdc-banner;
    @extend .wdc-banner-focus;
```

File: modules/css/button/lib/accessibility-deprecated.scss [hunk 2]
```diff
@@ -14,18 +14,18 @@
   }
 
   .wdc-btn-deprecated.wdc-btn-primary:not([disabled]):not(.wdc-btn-disabled):not([tabindex='-1']):focus {
-    @extend .wdc-btn-primary.wdc-btn-focus;
+    @extend .wdc-btn-primary, .wdc-btn-focus;
```
Comment (alanbsmith) [inline modules/css/button/lib/accessibility-deprecated.scss:17]:
```suggestion
    @extend .wdc-btn-primary;
    @extend .wdc-btn-focus;
```

File: modules/css/button/lib/accessibility-deprecated.scss [hunk 3]
```diff
@@ -14,18 +14,18 @@
...
   }
 
   .wdc-btn-deprecated.wdc-btn-split-text:not([disabled]):not(.wdc-btn-disabled):not([tabindex='-1']):focus {
-    @extend .wdc-btn-split-text.wdc-btn-focus;
+    @extend .wdc-btn-split-text, .wdc-btn-focus;
```
Comment (alanbsmith) [inline modules/css/button/lib/accessibility-deprecated.scss:21]:
```suggestion
    @extend .wdc-btn-split-text;
    @extend .wdc-btn-focus;
```

File: modules/css/button/lib/accessibility-deprecated.scss [hunk 4]
```diff
@@ -14,18 +14,18 @@
...
   }
   .wdc-btn-deprecated.wdc-btn-split-icon:not([disabled]):not(.wdc-btn-disabled):not([tabindex='-1']):focus {
-    @extend .wdc-btn-split-icon.wdc-btn-focus;
+    @extend .wdc-btn-split-icon, .wdc-btn-focus;
```
Comment (alanbsmith) [inline modules/css/button/lib/accessibility-deprecated.scss:24]:
```suggestion
    @extend .wdc-btn-split-icon;
    @extend .wdc-btn-focus;
```

File: modules/css/button/lib/accessibility-deprecated.scss [hunk 5]
```diff
@@ -14,18 +14,18 @@
...
   }
 
   .wdc-btn-deprecated.wdc-btn-delete:not([disabled]):not(.wdc-btn-disabled):not([tabindex='-1']):focus {
-    @extend .wdc-btn-delete.wdc-btn-focus;
+    @extend .wdc-btn-delete, .wdc-btn-focus;
```
Comment (alanbsmith) [inline modules/css/button/lib/accessibility-deprecated.scss:28]:
```suggestion
    @extend .wdc-btn-delete;
    @extend .wdc-btn-focus;
```

File: modules/css/button/lib/accessibility.scss [hunk 6]
```diff
@@ -19,7 +19,7 @@
   }
 
   .wdc-btn-primary:not([disabled]):not(.wdc-btn-disabled):not([tabindex='-1']):focus {
-    @extend .wdc-btn-primary.wdc-btn-focus;
+    @extend .wdc-btn-primary, .wdc-btn-focus;
```
Comment (alanbsmith) [inline modules/css/button/lib/accessibility.scss:22]:
```suggestion
    @extend .wdc-btn-primary;
    @extend .wdc-btn-focus;
```

File: modules/css/button/lib/accessibility.scss [hunk 7]
```diff
@@ -66,10 +66,10 @@
 
   .wdc-btn-delete:not([disabled]):not(.wdc-btn-disabled):not([tabindex='-1']) {
     &:hover {
-      @extend .wdc-btn-delete.wdc-btn-hover;
+      @extend .wdc-btn-delete, .wdc-btn-hover;
     }
     &:active {
-      @extend .wdc-btn-delete.wdc-btn-active;
+      @extend .wdc-btn-delete, .wdc-btn-active;
```
Comment (alanbsmith) [inline modules/css/button/lib/accessibility.scss:72]:
```suggestion
      @extend .wdc-btn-delete;
      @extend .wdc-btn-active;
```

File: modules/css/select/lib/select.scss [hunk 8]
```diff
@@ -55,7 +55,7 @@
 }
 
 [data-whatinput='keyboard'] .wdc-form-select:not([disabled]):focus {
-  @extend .wdc-form-select.wdc-form-focus;
+  @extend .wdc-form-select, .wdc-form-focus;
```
Comment (alanbsmith) [inline modules/css/select/lib/select.scss:58]:
```suggestion
  @extend .wdc-form-select;
  @extend .wdc-form-focus;
```

**Pull request merged**